### ISSUE TITLE
ONIX edit note

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#19]
From EditEUR's feedback:

# NOTE (Revisions)
ONIX does not have a 1:1 mapping with EPUB or Schema.org accessibility metadata. Some accessibility assertions are made differently, and some may not currently be possible . This EPUB to ONIX crosswalk outlines the current **high degree of overlap** in metadata and will be updated as these two specifications evolve. It is important to note that ONIX 3.0 includes a number of new accessibility metadata codes. <del>ONIX 2.1 can carry all the values from list 196 but has more limited accessibility metadata for other values and is not covered in this document.</del> **And not all may be carried in earlier versions of ONIX.**

**Comment by madeleinerothberg** [comment]
Proposed small edit to merge last two sentences, after the strikeout text is removed:

It is important to note that ONIX 3.0 includes a number of new accessibility metadata codes some of which may not be carried in earlier versions of ONIX.

**Comment by mattgarrish** [comment]
Will this work as an alternative?

ONIX does not have a 1:1 mapping with [EPUB accessibility metadata](http://idpf.org/epub/a11y/). Some accessibility assertions are made differently, and some may not currently be possible. The EPUB to ONIX crosswalk outlines the current high degree of overlap in metadata and will be updated as these two specifications evolve. It is important to note that ONIX 3.0 includes a number of new accessibility metadata codes, some of which may not be expressible in earlier versions of ONIX.

**Comment by BenSchroeter** [comment]
"Some accessibility assertions are made differently, and some may not currently be possible."

This wording may be confusing without context. Perhaps: 

"As some accessibility assertions are posed differently between the two mechanisms, direct mapping between the two may not currently be possible."

**Comment by mattgarrish** [comment]
Works for me!

**Comment by gregoriopellegrino** [comment]
Feedback from EditEUR:

> Yes - we wanted to just make sure that those who are unfamiliar with ONIX were aware that there is not always a one to one mapping as we are talking about metadata in the supply chain and metadata int he product itself and I think the wording you have used is good.
> Only one word I would change 
> "direct mapping between the two may not **currently** be possible"   I would suggest ‘**always**’ instead of ‘**currently**’ - as this makes it clearer that there is a lot of overlap but there may be certain areas where the approach is different
